deltachat-server/Cargo.toml

```diff
@@ -0,0 +1,26 @@
+[package]
+name = "deltachat-server"
+version = "1.98.0"
+description = "DeltaChat JSON-RPC server"
+authors = ["Delta Chat Developers (ML) <delta@codespeak.net>"]
+edition = "2021"
+readme = "README.md"
+license = "MPL-2.0"
+
+keywords = ["deltachat", "chat", "openpgp", "email", "encryption"]
+categories = ["cryptography", "std", "email"]
+
+[[bin]]
+name = "deltachat-server"
+
+[dependencies]
+deltachat-jsonrpc = { path = "../deltachat-jsonrpc" }
+
+anyhow = "1"
+env_logger = { version = "0.9.1" }
+futures-lite = "1.12.0"
+log = "0.4"
+serde_json = "1.0.85"
+serde = { version = "1.0", features = ["derive"] }
+tokio = { version = "1.21.2", features = ["io-std"] }
+yerpc = { version = "^0.3.1", features = ["anyhow_expose"] }
```

Comment:
why caret notation here?  that's exactly the same as the others and the others don't use the explicit notation.  its a bit confusing writing the same thing in two different ways